php/driftphp/composer.json

```
@@ -0,0 +1,33 @@
+{
+    "name": "drift/skeleton",
+    "description": "Skeleton repository ofr DriftPHP",
+    "type": "project",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Marc Morera",
+            "email": "yuhu@mmoreram.com"
+        }
+    ],
+    "require": {
+        "php": "^7.3",
```

does `driftphp` support php 8 ?

---

Yes, should.

---

The base docker image is still on top of php7.4. As soon as is updated, I can update here the image.

---

The image is created on-the-fly 😛  
Could you set something like
```suggestion
        "php": "~7.4.0|~8.0.0",
```
we need to unify this constraint for all php frameworks, but could be in an other `PR`

I'm not very safe with the `^` constraint, despite it is the default in composer

---

This will only work if the composer install is done with `--ignore-platform-reqs` flag, basically because some dependencies might require **only** php^7 and that would make composer install crash.